python/tvm/relay/frontend/tflite.py

```diff
@@ -704,6 +705,33 @@ def convert_conv(self, op, conv_type):
 
         return out
 
+    def convert_split(self, op):
+        """split implementation."""
+        try:
+            from tflite.BuiltinOptions import BuiltinOptions
+            from tflite.Operator import Operator
+            from tflite.SplitOptions import SplitOptions
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+
+        assert len(input_tensors) == 2, "input tensors length should be == 2"
+
+        input_tensor = input_tensors[1]
+        input_tensor_idx = input_tensor.tensor_idx
+
+        assert op.BuiltinOptionsType() == BuiltinOptions.SplitOptions
+
+        in_expr = self.get_expr(input_tensor_idx)
+        op_options = op.BuiltinOptions()
+        split_options = SplitOptions()
+        split_options.Init(op_options.Bytes, op_options.Pos)
+        num_splits = split_options.NumSplits()
+        retval = _op.split(in_expr, num_splits, axis=0)
```

Comment:
axis shouldn't be 0 explicitly. You should use:
```python
split_axis_tensor = input_tensors[0]
split_axis = self.get_tensor_value(split_axis_tensor)
```
Then pass `split_axis` to `_op.split(in_expr, num_splits, axis=split_axis)`

Reply:
Thanks. fixed now . Not sure why I ran into issues with it earlier.